backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/appsearchmodal/internal/BpkSection.kt

```diff
@@ -70,17 +72,32 @@ internal fun BpkSectionItem(item: BpkItem, modifier: Modifier = Modifier) {
             .fillMaxWidth()
             .clickable(onClick = item.onItemSelected),
         verticalAlignment = Alignment.CenterVertically,
-        horizontalArrangement = Arrangement.spacedBy(BpkSpacing.Base),
+        horizontalArrangement = Arrangement.SpaceBetween,
```

Comment:
not sure that's correct as that evenly spaces the items - instead of creating another row you can give the column a weight of 1, so it expands